README.md

```diff
@@ -1,7 +1,7 @@
 |                         | Build Status |
 |-------------------------|--------------|
-| master                  | [![C++ CI for OpenStudioApplication - master](https://github.com/openstudiocoalition/OpenStudioApplication/workflows/C++%20CI%20for%20OpenStudioApplication/badge.svg?branch=master)](https://github.com/openstudiocoalition/OpenStudioApplication/actions?query=branch%3Amaster)|
-| develop                 | [![C++ CI for OpenStudioApplication - develop](https://github.com/openstudiocoalition/OpenStudioApplication/workflows/C++%20CI%20for%20OpenStudioApplication/badge.svg?branch=develop)](https://github.com/openstudiocoalition/OpenStudioApplication/actions?query=branch%3Adevelop)      |
+| master                  | [![C++ CI for OpenStudioApplication - master](https://github.com/openstudiocoalition/OpenStudioApplication/actions/workflows/app_build.yml/badge.svg?branch=master)](https://github.com/openstudiocoalition/OpenStudioApplication/actions/workflows/app_build.yml?query=branch%3Amaster)|
```

Comment:
Was this pointing at a stale badge or something?  The link still returned a green svg